universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -156,6 +157,13 @@ public FieldWithLabel setValue(String value) {
     return this;
   }
 
+  public FieldWithLabel setValue(Optional<String> value) {
```

Comment:
suggestion (non-blocking): Might be good to `throw` if it's a `number` or `checkbox` type. But at this point I'm thinking it might make sense to put that in a small helper method like `assertStringField` or something?

Reply:
Good catch - I learned the hard way that "value"s can exist on checkboxes, so I just threw if the type is `number`